cryptocurrency/script.py

```diff
@@ -0,0 +1,54 @@
+from deephaven import jpy
+import os
+os.system("pip install pycoingecko") #install the package to use
+from pycoingecko import CoinGeckoAPI
+from deephaven.TableTools import merge
+from time import sleep
+from deephaven.DBTimeUtils import convertDateTime, secondsToTime, millisToTime
+import threading
+import time
+
+timeToWatch = 2 # minutes to keep table updating
+secondsToSleep = 10 # should be 10 or higher. If too fast will hit request limit
+
+getHistory = True
+daysHistory = 90 # if getHistory = true the days to pull
+
+#coins to get data on so far
+ids=['bitcoin', 'ethereum','litecoin', 'dogecoin', 'tether', 'binancecoin','cardano', 'ripple', 'polkadot']
+
+#tables to merge current and previous data
+tableArray=[]
+
+DynamicTableWriter = jpy.get_type("io.deephaven.db.v2.utils.DynamicTableWriter")#org.joda.time.base.BaseDateTime works
+tableWriter = DynamicTableWriter(["Coins", "DateTime", "prices", "market_caps", "total_volumes"], [jpy.get_type("java.lang.String"), jpy.get_type("io.deephaven.db.tables.utils.DBDateTime"), jpy.get_type("double"), jpy.get_type("double"), jpy.get_type("double")])
```

Comment:
Maybe add something to the README that explains what the columns mean.

Reply:
Thanks! done!